### PR TITLE
Regression test

### PR DESCRIPTION
**Pull request by qiliRedHat**
Tested with job on my own branch https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/qili-e2e-benchmark/job/regression-test/107/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Perf&Scale Tegression Test Job
```
Comment (paigerube14) [inline README.md:1]: Tegression -> Regression

File: README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+This job is used to trigger Perf&Scale regression test cases.
+### Parameters
+
+| Variable         | Description                         | Default |
+|------------------|-------------------------------------|---------|
+| **BUILD_NUMBER** | Flexy install build number          |         |
+| **TEST_CASER**   | Test case name. This job will search the <TEST_CASE>.sh file under svt repo <a href="https://github.com/openshift/svt/blob/master/perfscale_regression_ci/scripts">perfscale_regression_ci/scripts</a> folder and sub folders. If SCRIPT is specified, TEST_CASER will be overwritten.|conc_job |
```
Comment (paigerube14) [inline README.md:8]: TEST_CASER -> TEST_CASE

Comment (paigerube14) [reply README.md:8]: Since we added the folders in the default example won't work correct?

File: Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+                  Relative path to the script of the TEST_CASE under <a href="https://github.com/openshift/svt">svt repo</a>.<br>
+                  e.g.<br>
+                  For large_network_policy test case: perfscale_regression_ci/scripts/network/large_network_policy.sh<br>
+                  If TEST_CASE is specified, SCRIPT will overwrite the TEST_CASE.
+                  </p>''')
+    string(name: 'PARAMETERS', defaultValue: '', description: '''<p>
+                  Parameter or an array of parameters to pass to the TEST_CASE script<p>
+                  e.g.<br>
+                  For conc_jobs: 100 500 1000 2000<br>
+                  For large_network_policy: 5000<br>
```
Comment (paigerube14) [inline Jenkinsfile:32]: I'm confused about these parameters. What does this set in the test case? Doesn't look like [large_network_policy](https://github.com/openshift/svt/blob/master/perfscale_regression_ci/scripts/network/large_network_policy.sh) takes in any command line parameters

File: README.md (hunk 2 above)
Comment (qiliRedHat) [reply README.md]: It should work. I find the <TEST_CASE>.sh under 'scripts' and it's sub-folders here
https://github.com/qiliRedHat/ocp-qe-perfscale-ci/blob/c572986d8ee489d52fd4dce7bf854ac80c62f484/Jenkinsfile#L98

File: Jenkinsfile (hunk 3 above)
Comment (paigerube14) [reply Jenkinsfile:32]: Sounds good!

File: README.md (hunk 2 above)
Comment (qiliRedHat) [reply README.md]: > TEST_CASER -> TEST_CASE

Done. Thanks!

File: README.md (hunk 1 above)
Comment (qiliRedHat) [reply README.md]: Done. Thanks!

File: Jenkinsfile (hunk 3 above)
Comment (paigerube14) [reply Jenkinsfile:32]: are we going to wait to merge this PR till those changes are in?

**Comment by qiliRedHat** [comment]
@paigerube14 Updated according to your comments and squashed the commits. Thanks!

File: Jenkinsfile (hunk 3 above)
Comment (qiliRedHat) [reply Jenkinsfile:32]: No, please go ahead to merge. I'll update update [large_network_policy](https://github.com/openshift/svt/blob/master/perfscale_regression_ci/scripts/network/large_network_policy.sh) tomorrow.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Perf&Scale Regression Test Job
+This job is used to trigger Perf&Scale regression test cases.
+### Parameters
+
+| Variable         | Description                         | Default |
+|------------------|-------------------------------------|---------|
+| **BUILD_NUMBER** | Flexy install build number          |         |
+| **TEST_CASE**   | Test case name. This job will search the <TEST_CASE>.sh file under svt repo <a href="https://github.com/openshift/svt/blob/master/perfscale_regression_ci/scripts">perfscale_regression_ci/scripts</a> folder and sub folders. If SCRIPT is specified, TEST_CASER will be overwritten.|conc_job |
```
Comment (paigerube14) [inline README.md:8]: another TEST_CASER reference here (needs to be TEST_CASE)

Comment (qiliRedHat) [reply README.md]: Done. Thanks!

Comment (paigerube14) [reply README.md]: /lgtm

**Pull request merged**